package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "packetery",
+  "version": "1.0.0",
+  "scripts": {
+    "compile-scss": "sass public/scss:public/css --no-source-map --style compressed",
+    "watch:sass": "sass --watch public/scss:public/css --no-source-map --style compressed"
```

Comment:
Přidal bych dev verzi příkazu s mapováním a map soubory bych dal do gitignore. Zjednodušší nám to vývoj.